intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/detector/intent_detection.py

```diff
@@ -16,10 +16,17 @@
 # limitations under the License.
 """Check the intent of the input user query with LLM."""
 
-from intel_extension_for_transformers.neural_chat.pipeline.plugins.prompt.prompt_template \
-    import generate_intent_prompt
+# from intel_extension_for_transformers.neural_chat.pipeline.plugins.prompt.prompt_template \

```

Comment:
why not using the existing generate_intent_prompt?

Reply:
It's for better performance of retrieval, maybe @XuhuiRen could explain this better than me.

Reply:
> It's for better performance of retrieval, maybe @XuhuiRen could explain this better than me.

It should follow the previous usage. I ask you to put it here just for the convenience of testing the askgm demo. I will fix it.

Reply:
okay, I misunderstood.